_posts/2021-01-25-solidity-developer-survey-2020-results.md

```diff
@@ -0,0 +1,359 @@
+
+![Survey Participants World Map](/img/2021/01/world.png)
+
+#### Residency
+
+⚠️ _Note that the fact that this survey has only been shared in English is an important factor to consider when interpreting results regarding language and country of residency._
+
+The respondents stated to live in Algeria, Argentina, Australia, Austria, Belgium, Brazil, Bulgaria, Canada, China, Costa Rica, Denmark, Finland, France, Germany, Ghana, Greece, Honduras, Hong Kong, India, Indonesia, Iran, Ireland, Israel, Italy, Japan, Lithuania, Malaysia, Mexico, Netherlands, New Zealand, Nigeria, Pakistan, Philippines, Poland, Portugal, Romania, Russia, Serbia, Singapore, South Africa, Spain, Switzerland, Taiwan, Thailand, Timor-Leste, Ukraine, United Kingdom and United States of America. 
+
+With roughly 20%, the big majority of respondents live in the United States of America, followed by India (8%) and Germany (7%).
+
+![Survey Participants List of Countries](/img/2021/01/residency.png)
+
+#### Language
+
+The developers speak a wide variety of different languages as their primary language, with English being by far the most popular one. German, Spanish, Russian and French follow with similar levels.
+
+![Primary Language](/img/2021/01/langprimary.png)
+
+With 86%, the vast majority of respondents speak English at work. 
+
+Almost 90% of respondents are okay with reading the Solidity documentation in English while roughly 10% would prefer it to be translated into their native language. Some of the respondents who chose "native language" stated English as their primary spoken language though, which results in 95% of participants preferring English for the documentation.
+
+Bear in mind that this survey has only been conducted in English, which heavily influences the outcome of this question. This result will hence not have too much influence on the revamped translation efforts for the Solidity documentation and we will further investigate this topic.
+
+![Language at Work](/img/2021/01/langspeak.png)
+
+
+![Documentation Language](/img/2021/01/documentationlang.png)
+
+#### Employment
+
+82% of all respondents stated that they are currently employed. 4% identified as students, while 14% stated to not work at the moment. 
+
+![Employment Status](/img/2021/01/employment.png)
+
+#### Industry
+
+The majority of respondents (68%) said they work in the technology industry. 14% work in the financial services industry. Less relevant, but also selected several times, were the energy sector, health care, creative, media & gaming and transportation.
+
+![Industry Overview](/img/2021/01/industry.png)
+
+#### Coding Experience
+
+The overall, non-Solidity-related, professional coding experience of respondents was impressive. 48% of respondents said they coded as part of their jobs for 6 years or more. Only roughly 7% stated that they have less than 1 year of coding experience. 
+
+![Professional Coding Experience](/img/2021/01/codingexperience.png)
```

Comment:
Again, if it's not too much of a hassle, everything except the "no coding experince" result could be stacked on the same bar because each answer "includes" all of the previous ones.

Reply:
I'm not sure what you're suggesting exactly? That there are only two bars? I think that would be quite hard to read

Reply:
Ok, was just a suggestion!